io-engine/src/lvs/lvs_error.rs

```diff
@@ -265,6 +270,9 @@ impl ToErrno for Error {
             Self::WipeFailed {
                 ..
             } => Errno::EINVAL,
+            Self::ResourceLockFailed {
```

Comment:
Missing conversion on mayastor_grpc v0 to tonic::Status

Reply:
fixed